### PR TITLE
Sync items nodes published state

### PR DESCRIPTION
**Pull request by arvida**
https://trello.com/c/mSmvaUGE/59-sync-publish-for-items-with-public-for-nodes

### CODE REVIEW

File: app/models/node.rb [hunk 1]
```diff
@@ -80,6 +80,29 @@ def activate
     self.status = 0
   end
 
+  def destroy_children
```
Comment (joeljunstrom) [inline app/models/node.rb:87]: this shit is scary =) but should it be recursive? This will not walk down the tree right?

Comment (arvida) [inline app/models/node.rb:87]: Everything under the node should be destroyed when its deleted. I guess one could collect a list of all the stuff should be deleted aswell.

Comment (joeljunstrom) [inline app/models/node.rb:87]: But since we are not calling destroy_children on the.. children will we remove everything? Am I making sense? =)

Comment (arvida) [inline app/models/node.rb:87]: Ah true

**Comment by arvida** [comment]
Confirmation view added.

**Pull request merged**